session.go

```diff
@@ -258,6 +269,11 @@ func (s *Session) RunAnalyzeQuery(ctx context.Context, stmt spanner.Statement) (
 }
 
 func (s *Session) runQueryWithOptions(ctx context.Context, stmt spanner.Statement, opts spanner.QueryOptions) (*spanner.RowIterator, *spanner.ReadOnlyTransaction) {
+	// If query options are used without a DirectedReadOption, any session-level
+	// DirectedReadOption will be overridden. Always include DirectedReadOption in
+	// opts.DirectedReadOptions when necessary.
+	opts.DirectedReadOptions = s.clientConfig.DirectedReadOptions
```

Comment:
What does this mean? If `opts.DirectedReadOptions` is nil, client-level config won't be used?

Reply:
I also misunderstood this point.
To be precise, it seems that client-level config only does not work with RW Tx.
Of course, directed reads are not supported with RW Tx, but I observed the following difference.
I think it might be easier to understand that RW Tx stops working when the --directed-read option is passed.
What do you think?

Directed read options: `client-level` only
```
$ ./spanner-cli -p data-db-demo -i test-instance -d test-db --directed-read asia-northeast2
Connected.
spanner> begin rw;
Query OK, 0 rows affected (0.02 sec)

spanner(rw txn)> select 1;
+---+
|   |
+---+
| 1 |
+---+
1 rows in set (0.91 msecs)
```

Directed read options: `client-level` + `query-level`
```
$ ./spanner-cli -p data-db-demo -i test-instance -d test-db --directed-read asia-northeast2
Connected.
spanner> begin rw;
Query OK, 0 rows affected (0.02 sec)

spanner(rw txn)> select 1;
ERROR: spanner: code = "InvalidArgument", desc = "Directed reads can only be performed in a read-only transaction."
```

Reply:
I decided not to pass this option on a per-query basis.
Instead, I have added a note to the README to clarify that it does not apply to RW Tx.